iree_tests/README.md

```diff
@@ -1,5 +1,39 @@
 # IREE Tests
 
+## Quickstart instructions
+
+```
+#!/bin/bash
```

Comment:
nit: add an explicit language when known to code blocks, to help markdown renderers apply syntax highlighting
```suggestion
```bash
#!/bin/bash
```